kolibri/plugins/video_mp4_render/assets/src/vue/index.vue

```diff
@@ -77,9 +77,18 @@
 
     methods: {
       getLangName(langCode) {
-        return langcodes.filter(
-          (lang) => lang.code === langCode
-        )[0].lang;
+        if (LangLookup[langCode]) {
+          return LangLookup[langCode].native_name;
+        }
+        return langCode;
+      },
+      isDefaultTrack(langCode) {
+        const shortLangCode = langCode.split('-')[0];
+        const shortGlobalLangCode = global.languageCode.split('-')[0];
```

Comment:
Can also get this from `require('vue').locale` to avoid referencing the global variable (would also future proof it a bit more if we ever do dynamic language switching in the frontend).